src/main.js

```diff
@@ -0,0 +1,165 @@
+// SPDX-License-Identifier: Apache-2.0
+
+import { resolve } from "path";
+import { downloadTool, extractTar, extractZip } from '@actions/tool-cache';
+import { addPath, setFailed, getInput } from '@actions/core';
+import { platform as osPlatform, arch as osArch } from 'node:os';
+
+
+async function getLatestVersion() {
+    // Note: The GitHub API is rate limited to 60 requests per hour for unauthenticated users.
+    // I am not quite sure that it is reliable to query the latest version from
+    // the GitHub API because of the rate limits.
+    let response = await fetch(`https://api.github.com/repos/ispc/ispc/releases/latest`);
```

Comment:
I don't know Javascript but it seems that there is `let` and `const` keywords (both can be used to declare variables in JavaScript). 
`let`: Allows you to reassign the variable after its initial declaration.
`const`: Does not allow reassignment. Once a value is assigned, it cannot be changed.

In this script most of the variables are `const`.

Reply:
It looks like the linter fixed it.